lib/diagnostic.js

```diff
@@ -8,21 +8,40 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
+  let lowNum = arguments[0];
 
+  for (let i = 0; i < arguments.length; i--) {
```

Comment:
The explanation of for loops today during the lesson helped a little bit too. I second guessed myself and assumed it had to be different.